.gitlab-ci.yml

```diff
@@ -582,6 +582,10 @@ changelog:
     # GITHUB_TOKEN for Github cli authentication
     - export GITHUB_TOKEN=${GITHUB_CLI_TOKEN}
     - cp CHANGELOG.md CHANGELOG.md.${CI_COMMIT_REF_NAME} # Saving the changelog for later
+    - THIS_REPO=$(echo ${GITHUB_REPO_URL} | cut -d '/' -f2)
```

Comment:
There is a gitlab variable called `$CI_PROJECT_NAME` storing exactly this information :)